src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Book.ttf);
+  color: #3d4e61;
+}
+
+.search-image {
+  position: relative;
+}
+
+.search-image::before {
+  content: "";
+  position: absolute;
+  background: url(images/Search.svg) no-repeat;
+  top: 25px;
+  bottom: 0;
+  left: 26px;
+  margin: auto 0;
+  width: 25px;
+  /* height: 25px; */
+}
+
+.search-bar-big {
+  margin-top: 20px;
+}
+
+.search-field-big {
+  /* margin-top: 20px; */
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
+  padding-left: 62px ;
+  border: 1px solid #e1e7ed;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  /* background: url(images/Search.svg) no-repeat left; */
+  /* padding: 26px 25px 0 26px; */
+}
+
+.search-field-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-field-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  @font-face {
+    font-family: Avenir;
+    src: url(fonts/Avenir-Heavy.ttf);
+  }
+}
+
+.search-bar-small {
+  margin-top: 20px;
+}
+
+.search-image-small {
+  position: relative;
+}
+   
+.search-image-small::before {
+  content: "";
+  position: absolute;
+  background: url(images/Search.svg) no-repeat;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+  margin: auto 0;
+  width: 11px;
+  height: 11px;
+  background-size: 11px;
+}
+
+.search-field-small {
+  height: 42px;
+  width: 100%;
+  border-radius: 4px;
+  padding-left: 33px;
+  border: 1px solid #e1e7ed;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  /* background: url(images/Search.svg) no-repeat left; */
+  /* padding: 26px 25px 0 26px; */
+}
+
+.search-field-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-field-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  @font-face {
+    font-family: Avenir;
+    src: url(fonts/Avenir-Heavy.ttf);
+  }
+}
```

Comment:
pay attention to the checklist and avoid duplicating

Reply:
cleaned up similar code